src/gui/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 project(gui)
 find_package(Qt5 REQUIRED COMPONENTS Widgets Svg Qml Quick)
+if (Qt5Widgets_VERSION VERSION_LESS 5.9.0)
```

Comment:
Why not simply pass the required version in the find_package call?
Granted you'd get a less fine tuned error message by default... but is that really necessary?

Mostly being curious here.

Reply:
BTW, it should likely be 5.10 and not 5.9 in that check since the icon property is used with the Button item.